build/compile/LzmaArchive.targets

```diff
@@ -79,6 +79,7 @@
         <FilesToClean Include="$(NuGetPackagesArchiveFolder)/**/*" />
 
         <NuGetPackagesArchiveDirectory Include="$(NuGetPackagesArchiveProject)/Console" />
```

Comment:
Do we still need a separate one for console now that we have the mvc one? I mean, is the MVC a super-set?